pages/plugins.rst

```diff
@@ -34,14 +49,26 @@ What you need in your development environment before starting is:
 
 There are lots of different ways to get those on your local machine, unfortunately we cannot list all of them, so please refer to your operating system-specific documentation,
 
+Graylog uses a couple of conventions and techniques in its code, so be sure to read about the :ref:`general_concepts_api` for an overview.
+
+.. _sample_plugin:
+
+Sample Plugin
+-------------
+
+To go along with this documentation, there is a `sample plugin on Github <https://github.com/Graylog2/graylog-plugin-sample/tree/2.2>`_. Tis documentation will link to specific parts for your reference.
+It is fully functional, even though it does not implement any real logic. Its purpose so to give a reference for helping to implement your own plugins.
+
 .. _creating_plugin_skeleton:
 
 Creating a plugin skeleton
-==========================
+--------------------------
 
 The easiest way to get started is to use our `Graylog meta project <https://github.com/Graylog2/graylog-project>`_,
 which will create a complete plugin project infrastructure will all required classes, build definitions, and configurations. Using the meta project allows you to have the `Graylog server project <https://github.com/graylog2/graylog2-server>`_ and your own plugins (or 3rd party plugins) in the same project, which means that you can run and debug everything in your favorite IDE or navigate seamlessly in the code base.
 
+.. note:: We are working on a replacement tool for the ``graylog-project`` meta project, but for the time being it still works.
```

Comment:
I fixed the script instead of getting rid of this section.

https://github.com/Graylog2/graylog-project/commit/31da1143446181269b36b44a6f4d1ae1be20d02e